packages/smart-wallet/src/smartWallet.js

```diff
@@ -65,18 +64,13 @@ const mapToRecord = map => Object.fromEntries(map.entries());
 
 /**
  * @typedef {{ updated: 'offerStatus', status: import('./offers.js').OfferStatus } |
- * { updated: 'balance'; currentAmount: Amount } |
- * { updated: 'brand', descriptor: BrandDescriptor }
+ *  { updated: 'balance'; currentAmount: Amount }
  * } UpdateRecord Record of an update to the state of this wallet.
  *
  * Client is responsible for coalescing updates into a current state. See `coalesceUpdates` utility.
  *
- * The reason for this burden on the client is that transferring the full state is untenable
- * (because it would grow monotonically).
- *
- * `balance` update supports forward-compatibility for more than one purse per
- * brand. An additional key will be needed to disambiguate. For now the brand in
- * the amount suffices.
```

Comment:
why lose this part?

Reply:
Because this PR gets rid of `balance` updates, no? hm... maybe not for zoe invitations...

Reply:
`balance` updates are still here, as they should be for Invitations
https://github.com/Agoric/agoric-sdk/blob/fac5c322aab4a7aa34aee5cfb04087dd4c63b252/packages/smart-wallet/src/smartWallet.js#L540-L541

Please restore the design note

Reply:
oops... forgot gain. sorry